### PR TITLE
New simulation stack versions

### PR DESCRIPTION
**Pull request by jorblancoa**
- Neuron 8.0b
 - CoreNeuron 1.0b
 - neurodamus-(mousify, neocortex, thalamus) 1.4
 - neurodamus-hippocampus 1.5
 - neurodamu-py 2.5.2
 - neurodamus-core 3.2.2
 - libsonata-report 0.1b

### CODE REVIEW

File: var/spack/repos/builtin/packages/coreneuron/package.py [hunk 1]
```diff
@@ -91,6 +92,7 @@ class Coreneuron(CMakePackage):
     # with the contemporary develop version it seems to work. Encode this
     # knowledge as a conflict between +report and +gpu for older versions.
     conflicts('+report', when='coreneuron@:1.0a+gpu+openmp')
+    conflicts('+report', when='coreneuron@:1.0b+gpu+openmp')
```
Comment (iomaganaris) [inline var/spack/repos/builtin/packages/coreneuron/package.py:94]: I think that `:1.0b` should cover also `:1.0a`? Have you tested if this is not working for some reason maybe?

File: var/spack/repos/builtin/packages/neurodamus-hippocampus/package.py [hunk 2]
```diff
@@ -20,7 +20,7 @@ class NeurodamusHippocampus(NeurodamusModel):
     # IMPORTANT: Register versions (only) here to make them stable
     # Final version name is combined e.g. "1.0-3.0.1"
     model_core_dep_v = (
-        ('1.4.1', '3.2.0'),
+        ('1.4.1', '3.2.2'),
```
Comment (iomaganaris) [inline var/spack/repos/builtin/packages/neurodamus-hippocampus/package.py]: Should we keep the previous version (`1.4.1-3.2.0`) as well? @ferdonline @pramodk

Comment (pramodk) [reply var/spack/repos/builtin/packages/neurodamus-hippocampus/package.py]: It's in archive. So I think its ok to rid of it and avoid confusion.

File: var/spack/repos/builtin/packages/coreneuron/package.py (hunk 1 above)
Comment (pramodk) [reply var/spack/repos/builtin/packages/coreneuron/package.py:94]: that's right!

**Review by pramodk — COMMENTED** [review]
Approve from Ionnias's minor comment, this LGTM.

@ferdonline : any comments? all py/hoc-neurodamus covered?

**Comment by jorblancoa** [comment]
retest this please

**Review by ferdonline — DISMISSED** [review]

**Review by ferdonline — APPROVED** [review]

**Pull request merged**